docs/spec/generics.rst

```diff
@@ -93,8 +93,13 @@ This is equivalent to omitting the generic notation and just saying
 User-defined generic types
 --------------------------
 
-You can include a ``Generic`` base class to define a user-defined class
-as generic.  Example::
+You can define a user-defined class as generic in three ways: by including a
```

Comment:
Should the case where `Generic` is optional be explicitly mentioned here? I mean this:
```py
from typing import TypeVar

T = TypeVar("T")

class A(list[T]): pass
```
Alternatively, this could rewritten so that it's clear that these three aren't the only ones.